sh.exe.stackdump

```diff
@@ -0,0 +1,22 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FF8D7F2AAB7
```

Comment:
I have tried to delete it twice. Fist time with regular delete, second time via TortoiseGit --> Delete. sh.exe.stackdump is created every time I run >>contribute in MATLAB. First time contribute (3) detected modifications (since regular delete removes file only locally and file is created again every time I suppose it is valid scenario to detect modifications not deletion), second time contribute (3) detected deletion and prompted  to accept deletion ( I entere "Y" in order to delete file) and after that prompted for adding back the same file ( I enter "N" in order to not add sh.exe.dumpfile), however it is back again in git. The last option I can think off right now is to add it to .gitignore file. Any suggestions?
Regards